zoompage.php

```diff
@@ -96,6 +96,9 @@ function zoomPage($id, $urlhash)
 	if ($hideinfo == false || $correcthash == true || User::getCurrent()->isAdmin() || User::getCurrent()->isCheckuser()) {
 		$smarty->assign("showinfo", true);
 	}
+
+	$emailDomain = explode("@", $request->getEmail())[1];
```

Comment:
`test"testy @ test"@example.com` is a valid email address IIRC. For that matter, I _believe_ `test@example.com(foo@bar)` is too.

Yes, email addresses are horrid. I don't know if we want to attack this specific scenario, but it's something to bear in mind.